server.ts

```diff
@@ -72,10 +74,12 @@ export function app() {
           `'self'`,
           'data:',
           'https:',
+          matomoHost,
```

Comment:
Is it right that the three items above are URI schemes but what you've added is a hostname? I'm not sure where the docs are on what's supposed to go in this list.

Reply:
It looks like both are allowed. https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP

I just went with host out of habit as it's documented that way more often / matches other places in this file. But obviously if we can be stricter about SSL we should, so I'll amend.

Reply:
Thanks, useful link.